core/modules/proxy/mysqlProxy.lua

```diff
@@ -708,23 +708,31 @@ function read_query_result(inj)
     -- we injected query with the id=1 (for messaging and locking purposes)
     if (inj.type == 1) then
         print("q1 - ignoring")
+        local error_msg = ""
         for row in inj.resultset.rows do
-           print("   " .. row[1] .. " " .. row[2] .. " " .. tostring(row[3]) .. " " .. row[4])
-           if (tonumber(row[2]) < 0) then -- errors have code < 0
+           if (row[4] == "ERROR") then
               queryErrorCount  = queryErrorCount + 1
```

Comment:
please put a comment near the place when you create schema for that table saying that we are sensitive to column positions.